static/js/click_handlers.js

```diff
@@ -514,6 +517,36 @@ export function initialize() {
         }
     });
 
+    $("body").on("click", "#all_private_messages_icon", (e) => {
+        e.preventDefault();
+        $("#private_messages").addClass("active_private_messages_section");
+        top_left_corner.deselect_top_left_corner_items();
+        window.location.href = "#narrow/is/private";
+    });
```

Comment:
I expect we can delete this click handler in favor of just an `a href="/#narrow/is/private"`, and that'd be cleaner.

Reply:
umm, I'm not sure whether we can do it, as in this -

I've added a click handler to perform 2 functions ->
1.) Add class of active-filter to PRIVATE MESSAGES section, which just highlights it.
2.) Also at the same time i need to redirect it to `#narrow/is/private` which i cannot perform if i add `e.preventDefault()` to this click handler.

Let me know if there's an alternative in your mind for this.